.coveragerc

```diff
@@ -1,4 +1,5 @@
 [run]
+source = _pytest
 omit = 
     # standlonetemplate is read dynamically and tested by test_genscript
```

Comment:
Oh this does not exist anymore and can be removed. 😁

Reply:
standlonetemplate?

Apart from that I think we should also include `testing` (the tests themselves).  Anything else?

Reply:
"standlonetemplate" was used by the removed `genscript` feature.

> Apart from that I think we should also include testing (the tests themselves). Anything else?

Not sure, then `xfail` tests which are never meant to run because they reproduce issues not yet fixed will drop the coverage. In my head "coverage" does not include the tests. What's your opinion?

Reply:
I think unexpected missing coverage is a good sign of something being wrong.
Those xfail tests should be marked with "pragma: no cover" then.